hyp3_autorift/netcdf_output.py

```diff
@@ -189,6 +189,7 @@ def netCDF_packaging(VX, VY, DX, DY, INTERPMASK, CHIPSIZEX, CHIPSIZEY, SSM, SSM1
     nc_outfile.setncattr('date_created', datetime.datetime.now().strftime("%d-%b-%Y %H:%M:%S"))
     nc_outfile.setncattr('title', title)
     nc_outfile.setncattr('autoRIFT_software_version', IMG_INFO_DICT["autoRIFT_software_version"])
+    nc_outfile.setncattr('autoRIFT_parameter_file', parameter_file.replace('/vsicurl/', ''))
```

Comment:
I'd consider having the calling function be responsible for stripping `/vsicurl/`, but I see this function is called at least six times in the other vendor scripts, so I'll concede that this way is simpler.  Not blocking.

Reply:
Oh yeah, way better to do this in `process.py`